### PR TITLE
Dell optiplex/acpi fixes

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: src/southbridge/intel/common/acpi/pcie_port.asl [hunk 1]
```diff
@@ -5,13 +5,69 @@
...
+	Else
+	{
+		HPCS = One
+		Return (0)
+	}
+}
+
+Name (_HPP, Package (0x04) {
+	0x08, 
+	0x40,
```
[the remark this answers is not in the document]
Comment (krystian-hebel) [reply src/southbridge/intel/common/acpi/pcie_port.asl:64]: Line 65 as well. Also, please update your pre-commit hooks as it should have caught it.

[the remark this answers is not in the document]
Comment (krystian-hebel) [reply src/southbridge/intel/common/acpi/pcie_port.asl:64]: So how does https://results.pre-commit.ci/run/github/316342590/1696336457.T4tESHnLQR-WcOZRlFSgvg report errors just for these lines and not existing ones? I thought it uses the same logic as local pre-commit.

[the remark this answers is not in the document]
Comment (krystian-hebel) [reply src/southbridge/intel/common/acpi/pcie_port.asl:64]: Oh, I see, it is another hook for `coreboot_lint` that catches it.

[the remark this answers is not in the document]
Comment (miczyg1) [reply src/southbridge/intel/common/acpi/pcie_port.asl]: Fixed https://github.com/Dasharo/coreboot/pull/416/commits/4a7eea2da20287b1cf2d6a7acb2a9c12190b3acd

File: src/southbridge/intel/bd82x6x/acpi/pch.asl [hunk 2]
```diff
@@ -256,3 +256,125 @@ Method (_OSC, 4)
...
+		}
+
+		If (((RP8D == Zero) && \_SB.PCI0.RP08.HPCS))
+		{
+			If (\_SB.PCI0.RP08.HPEV () == 1) {
+				Notify (\_SB.PCI0.RP08, 0)
+			}
+		}
+	}
+}
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply src/southbridge/intel/bd82x6x/acpi/pch.asl]: Fixed https://github.com/Dasharo/coreboot/pull/416/commits/4a7eea2da20287b1cf2d6a7acb2a9c12190b3acd

**Comment by krystian-hebel** [comment]
Code looks fine, I'll approve after testing it on hardware.

**Review by krystian-hebel — APPROVED** [review]
Confirm, CPU usage dropped to reasonable values.

**Pull request merged**